src/ossl/rsa.rs

```diff
@@ -147,34 +149,69 @@ fn object_to_rsa_private_key(key: &Object) -> KResult<EvpPkey> {
     EvpPkey::from_ptr(pkey)
 }
 
-macro_rules! name_to_vec {
-    ($name:expr) => {
-        unsafe {
-            slice::from_raw_parts(
-                $name.as_ptr() as *const std::os::raw::c_char,
-                $name.len(),
-            )
-            .to_vec()
-        }
-    };
+fn mgf1_to_digest_buf(mech: CK_MECHANISM_TYPE) -> &'static [u8] {
```

Comment:
well I wanted to reserve name for when we return a c_char *, and here we return a u8 slice.
Would mgf1_to_digest_as_slice() work better for you ?

Reply:
My point was that `digest buffer` is something I would expect to be the buffer with the digest, rather than the `digest name as a buffer`. So my preference would be to have `mgf1_to_digest_name_as_slice` or `mgf1_to_digest_name_buf` or something in between.

Reply:
ok fixed